CHANGELOG.md

```diff
@@ -18,6 +18,10 @@ Implement Yes/No reading functions:
   * Add `YesNo` type (`Yes` | `No`)
 
   (by [@martinhelmer])
+  
+[#116](https://github.com/chshersh/iris/issues/116):
+Supports GHC 9.6.1
+  (by [@celsobonutti])
```

Comment:
Minor formatting changes.

Also, could you please your GitHub username to the `<!-- Contributors -->` section further below in this file?

```suggestion
* [#116](https://github.com/chshersh/iris/issues/116):
  Supports GHC 9.6.1
  (by [@celsobonutti])
```